### PR TITLE
fix: add link to new fastify website repository on organisations page

### PR DESCRIPTION
**Pull request by HassanBahati**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: versions-shipped.json [hunk 1]
```diff
@@ -1 +1,156 @@
-[]
+[
+  "1.0.0",
```
Comment (HassanBahati) [inline versions-shipped.json:2]: @Eomm these version were generated when i ran `npm run build:website` 

wondering if i missed something to ignore them....or they are supposed to be removed manually

File: src/pages/organisations.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ Is your organisation using Fastify? That's great!
 
 If you would like your organisation to have its logo featured in this list, here's how you do it! You can submit your organisation, your logo and other few relevant details by following these steps:
 
-1. Fork the [Fastify website repository](https://github.com/fastify/website) on GitHub
+1. Fork the [Fastify website repository](https://github.com/fastify/website-next) on GitHub
```
Comment (climba03003) [inline src/pages/organisations.mdx:20]: Not sure, I would assume this repo would change the name to `website` once it's done. 
And the old one would keep with another name.

Comment (HassanBahati) [reply src/pages/organisations.mdx:20]: current [Fastify website repository](https://github.com/fastify/website) doesnt have [this directory](https://github.com/fastify/website-next/tree/main/static) which that guide is addressing
https://github.com/fastify/website-next/blob/4fa3897759fc82cd1cfbf5f7da459fd75b8c2abb/src/pages/organisations.mdx?plain=1#L21
which gets confusing to current visitors to this website

Comment (Eomm) [reply src/pages/organisations.mdx:20]: > which gets confusing to current visitors to this website

I think this is the point: maybe we should add a banner on this website that stands "🚧 work in progress".
Right now fastify.io is the source of truth - an we will need to align once again the organizations as well before "going to production" 

But in general, I would reason that `website-next` will be renamed for sure to `website`

File: versions-shipped.json (hunk 1 above)
Comment (Eomm) [reply versions-shipped.json:2]: > wondering if i missed something to ignore them....or they are supposed to be removed manually

Right now they should be removed manually from the commit - will find a better approach with #46

**Review by Eomm — COMMENTED** [review]
After working on CI&CD I think we are not going to rename the repo (at least in the immediate future).

So I think this PR is valuable and should be merged 👍🏼 

@HassanBahati could you revert the changes to the `*.json` files so we can land it?

**Comment by HassanBahati** [comment]
> @HassanBahati could you revert the changes to the `*.json` files so we can land it?

yah sure, let me do so

**Review by Eomm — APPROVED** [review]

**Pull request merged**